.github/workflows/deploy-cdn.yml

```diff
@@ -0,0 +1,41 @@
+name: Deploy CDN Proxy
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'cdn/**/*'
+
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
```

Comment:
ha I actually viewed that the reverse light! That having us on latest would keep us ahead of failures for our end users

Reply:
Sure, but it's less likely that all of our users will rush into a new LTS version overnight; the a majority of our users will likely graduate into a new LTS at their comfort.  The glory of pinning it to a version is that we slowly receive the pressure from bleeding-edge users to address it, rather than inadvertently releasing a broken version to _all_ users just because a new LTS was published.

Anyhow, this doesn't matter at all because from what I can tell, this doesn't do the actual compiling?